packages/itwin/grouping-mapping-widget/src/widget/components/utils.tsx

```diff
@@ -106,3 +106,8 @@ export const LoadingSpinner = () => {
     </div>
   );
 };
+
+export const enableExperimentalFeatures = async (iModelConnection: IModelConnection) => {
+  const reader = iModelConnection.createQueryReader("PRAGMA experimental_features_enabled=true");
```

Comment:
I think this will no longer be required when iTwinJs 4.2 will come out: https://github.com/iTwin/imodel-native/pull/451